### PR TITLE
List bcc projects

### PR DESCRIPTION
**Pull request by kkuepper**
create a list of bcc projects and its status

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -11,9 +11,18 @@ description: Technical documentation and guides for software development in BCC
 
 * Check out the rest of this site for documentation.
 
+## Projects & Status
+* BMM: Open Source: [APP](https://github.com/bcc-code/bmm-app), [Old Website](https://github.com/bcc-code/bmm-website) and in active development
+* BrunstadTV: [App](https://github.com/bcc-code/brunstadtv-app), [Backend](https://github.com/bcc-code/brunstadtv) and in active development
+* Literature: [Open Source](https://github.com/bcc-code/bcc-literature) and in maintenance mode
```
Comment (rvanoord) [inline index.md:17]: and currently in maintenance mode

File: index.md [hunk 2]
```diff
@@ -11,9 +11,18 @@ description: Technical documentation and guides for software development in BCC
...
+* User Voice: [Open Source](https://github.com/bcc-code/uservoice) and in maintenance mode
+* Active Christianity: partly [Open Source](https://github.com/bcc-code/bcc-ac-gatsby-v2)
+* MyShare: currently closed source, but planned to be open sourced sometime in 2023
+* SongTreasures: currently closed source and no plans to change it
```
Comment (rvanoord) [inline index.md:21]: SongTreasures: maintained by Stiftelsen Skjulte Skatters Forlag

Comment (kkuepper) [reply index.md]: done

File: index.md (hunk 1 above)
Comment (kkuepper) [reply index.md]: changed

**Review by rvanoord — APPROVED** [review]

**Pull request merged**